assembly/fabric8-ide-dashboard-war/src/assets/branding/product.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "Eclipse Che ",
+  "name": "Eclipse Che",
+  "logoFile": "che-logo.svg",
+  "logoTextFile": "che-logo-text.svg",
+  "favicon": "favicon.ico",
+  "loader": "loader.svg",
+  "websocketContext": "/api/websocket",
+  "helpPath": "https://www.eclipse.org/che/",
+  "helpTitle": "Community",
+  "supportEmail": "che-dev@eclipse.org",
+  "oauthDocs": "Configure OAuth in the che.properties file.",
+  "workspace": {
+    "priorityStacks": ["Java", "Java-MySQL", "Blank"],
+    "defaultStack": "java-mysql"
+  },
+  "cli" : {
+    "configName" : "che.env",
+    "name" : "CHE"
+  },
+  "docs" : {
+    "devfile" : "/docs/che-7/using-developer-environments-workspaces.html#making-a-workspace-portable-using-a-devfile_using-developer-environments-workspaces",
+    "workspace" : "/docs/che-7/workspaces-overview/",
+    "factory" : "/docs/factories-getting-started.html",
+    "organization": "/docs/organizations.html",
+    "converting": "/docs/che-7/converting-a-che-6-workspace-to-a-che-7-devfile/",
+    "general": "/docs/che-7",
+    "faq": "https://github.com/redhat-developer/rh-che/blob/master/FAQ.adoc"
+  },
+  "configuration": {
+    "menu": {
+      "disabled": ["organizations"]
+    }, 
+    "features": {
+      "disabled": ["workspaceSharing","kubernetesNamespaceSelector"]
+    },
+    "prefetch": {
+      "cheCDN":"/api/cdn-support/paths"
```

Comment:
@davidfestal could you please comment which one should be used here?

Reply:
wondering if we still need https://github.com/redhat-developer/rh-che/tree/master/plugins/fabric8-cdn-support/src/main/java/com/redhat/che/cdn ?

Reply:
I assume we still need it yes. I'm not aware of this having been migrated to the upstream Che server codebase.
And the URL should be the same as well.

Reply:
@davidfestal do we need to pre-fetch some `resoures` also ?
https://github.com/eclipse/che/pull/15889

Reply:
I don't think so